samples/CMakeLists.txt

```diff
@@ -84,6 +84,22 @@ if (CONFIG_NCS_INCLUDE_RPMSG_CHILD_IMAGE OR CONFIG_NCS_SAMPLE_EMPTY_NET_CORE_CHI
       "couldn't resolve which RPMsg sample to add.")
   endif()
 
+  if (CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU)
+    # By passing this value we don't require the user to pass any overlay files
+    # to the MCUboot child image for storing the secondary partition in
+    # external flash.
+    if (CONFIG_NCS_SAMPLE_HCI_RPMSG_CHILD_IMAGE OR CONFIG_NCS_SAMPLE_MULTIPROTOCOL_RPMSG_CHILD_IMAGE)
+      add_overlay_config(
```

Comment:
@tejlmand needs to OK this, we don't really want more child/parent code parts added as we're trying to move to sysbuild

Reply:
@tejlmand, please take a look